presto-docs/src/main/sphinx/admin/resource-groups-example.json

```diff
@@ -39,12 +39,12 @@
               ]
             },
             {
-              "name": "bi-${tool_name}",
+              "name": "bi-${toolname}",
```

Comment:
Is this just a rename or do underscores not work?

Reply:
Undescore is not valid in Pattern's capturing group name (`java.util.regex.Pattern#groupname`) nor in ResourceGroupNameTemplate variable name: https://github.com/prestosql/presto/blob/6c0ff741af889f4434bdf2c28af155a5ee7354c1/presto-resource-group-managers/src/main/java/io/prestosql/plugin/resourcegroups/ResourceGroupNameTemplate.java#L31